lib/rubocop/cop/style/numbered_parameters_limit.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop detects use of an excessive amount of numbered parameters in a
+      # single block. Having too many numbered parameters can make code too
+      # cryptic and hard to read.
+      #
+      # The cop defaults to registering an offense if there is more than 1 numbered
+      # parameter but this maximum can be configured by setting `Max`.
+      #
+      # @example
```

Comment:
Adding the config parameter to the following example may help the user understand :-)

```suggestion
      # @example Max: 1 (default)
```

Reply:
Another good suggestion! :grin: